pyface/ui/wx/tasks/editor_area_pane.py

```diff
@@ -0,0 +1,176 @@
+# Standard library imports.
+import sys
+import logging
+
+# Enthought library imports.
+from pyface.tasks.i_editor_area_pane import IEditorAreaPane, \
+    MEditorAreaPane
+from traits.api import on_trait_change, provides
+
+# System library imports.
+import wx
+from pyface.wx.aui import aui, PyfaceAuiNotebook
+
+# Local imports.
+from task_pane import TaskPane
+
+# Logging
+logger = logging.getLogger(__name__)
+
+
+###############################################################################
+# 'EditorAreaPane' class.
+###############################################################################
+
+@provides(IEditorAreaPane)
+class EditorAreaPane(TaskPane, MEditorAreaPane):
+    """ The toolkit-specific implementation of a EditorAreaPane.
+
+    See the IEditorAreaPane interface for API documentation.
+    """
+
+    style = aui.AUI_NB_WINDOWLIST_BUTTON|aui.AUI_NB_TAB_MOVE|aui.AUI_NB_SCROLL_BUTTONS|aui.AUI_NB_CLOSE_ON_ACTIVE_TAB
+
+    ###########################################################################
+    # 'TaskPane' interface.
+    ###########################################################################
+
+    def create(self, parent):
+        """ Create and set the toolkit-specific control that represents the
+            pane.
+        """
+        logger.debug("editor pane parent: %s" % parent)
+        # Create and configure the tab widget.
+        self.control = control = PyfaceAuiNotebook(parent, agwStyle=self.style)
+#        control.tabBar().setVisible(not self.hide_tab_bar)
```

Comment:
This and the code below are presumably from attempts to hide/show the tab bar?  Do you want to keep around for reference, or can they be removed?